manifests/server/initdb.pp

```diff
@@ -82,11 +83,17 @@
       $require_before_initdb = [$datadir]
     }
 
-    $initdb_command = $locale ? {
+    $ic_locale = $locale ? {
```

Comment:
what is the meaning of this variable name, it is a little terse. was there any reason to rename it ?